10up/simple-google-news-sitemap
# Fix potential for Sitemap URLs to 404

### Description of the Change


Updates the `Core::add_sitemap_robots_txt()` method to account for two things:

* uses `home_url()` to avoid 404s in WordPress sub-directory installs using pretty permalinks
* checks for a permalink structure and displays the plain permalink if it is not set.



Closes #23

### How to test the Change


For sites running wordpress in a sub-directory mode, please see the steps in #23

For sites with plain permalinks:

1. Visit the Dashboard > Settings > Permalinks page
2. Set permalink structure to plain
3. Visit http://yourwordpresssite.example.com/?robots=txt
4. Verify the link uses the querystring parameter
5. Copy & paste link to verify it works.

### Changelog Entry

> Fixed - Potential 404 for sitemap URL added to robots.txt.


### Credits

Props @peterwilsoncc 


### Checklist:
